docs/user_guide.md

```diff
@@ -248,3 +249,75 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
+![browse project](img/scan_menu_item.png)
+**NOTES: Only the users with 'Project Admin' role have the accesses to trigger the analysis process.**
+
+The analysis process may have the following status that are indicated in the 'Vulnerability' column:
+* **Not Scanned:** The tag is never scanned so far.
+* **Queued:** The task is scheduled but not executed yet.
+* **Scanning:** The process is in progress.
+* **Error:** The process is failed to complete.
+* **Complete:** The process is successfully done.
+
+For the **'Not Scanned'** and **'Queued'** statuses, a text label with status information is shown. For the **'Scanning'**, a progress bar will be displayed.
+If error occurred, you can click the **'View Log'** link to open the log viewer to check the related logs.
+![browse project](img/log_viewer.png)
+
+If process is successfully completed, a result bar chart is created. The width of the different colors are used to indicate the percent of features with different severity vulnerabilities found in the overall features. 
+* **Color Red:** High level vulnerabilities
+* **Color Orange:** Medium level vulnerabilities
+* **Color Yellow:** Low level vulnerabilities
+* **Color Grey:** Unknown level vulnerabilities
+* **Color Green:** None vulnerabilities
+![browse project](img/bar_chart.png)
+
+Move cursor over the bar chart, a tooltips with summary report will displayed. Besides telling you the total features with vulnerabilities and the total count of features in the scanned tag, the report also lists the counts of features with different severity vulnerabilities. The completion datetime of the last analysis process is also appended at the footer of the tooltips.
+![browse project](img/summary_tooltip.png)
+
+Click the tag name link, the detail page will be opened. Besides more detailed information about the tag itself, all the vulnerabilities found in last analysis process will be listed here with the related information. You can order or filter the list by columns.
+![browse project](img/tag_detail.png)
+
+**NOTES: You can trigger the vulnerability analysis action for tag at anytime you want as long as the status is not 'Queued' or 'Scanning'** 
+
+**Scan All**
+
+Go to **'Vulnerability'** tab of **'Configuration'** section under **'Administration'**, you'll find the **'SCAN NOW'** button. Click that button will start the analysis process for all the existing images. 
+
+**NOTES: The process above is executed via multiple concurrent asynchronous tasks and then no guarantee to the order of executing or the results being returned.** 
+![browse project](img/scan_all.png)
+
+To avoid frequent triggering this 'heavy' action, the availability of the button will be restricted. It can be only triggered once in a specified duration. The next available time will be displayed as tips aside the button.

```

Comment:
To avoid frequently triggering the resource intensive scanning process, the availability of the button is restricted. It can be only triggered once in a predefined period. The next available time will be displayed besides the button.